src/viur_cli/cloud.py

```diff
@@ -0,0 +1,574 @@
+import json
+import subprocess
+import os
+import string
+import click
+import yaml
+from viur_cli import echo_positive, echo_warning, echo_fatal
+from .conf import config
+from . import cli, echo_error, echo_info, replace_vars
+from .update import create_req
+
+
+@cli.group()
+def cloud():
+    """Manage cloud Actions"""
+
+
+@cloud.command(context_settings={"ignore_unknown_options": True})
+@click.argument("action", type=click.Choice(["backup"]))
+def enable(action):
+    """
+        Enable specific features for the project.
+    """
+    if action == "backup":
+        enable_gcp_backup()
+
+
+def enable_gcp_backup():
+    # Load the project Config
+    conf = config.get_profile("default")
+
+    # Create helper Variables
+    project_id = conf["application_name"]
+    bucket_name = f'backup-dot-{project_id}'
+    backup_bucket_command = f'gsutil mb -l EUROPE-WEST3 -p {project_id} gs://{bucket_name}'
+
+    # Create the Backup Bucket
+    try:
+        result = subprocess.run(
+            backup_bucket_command,
+            capture_output=True,
+            shell=True,
+        )
+        print(result)
+        if result.returncode != 0:
+            print('Error creating bucket.')
```

Comment:
Use the cli-output utilities.
```suggestion
            echo_error('Error creating bucket.')
```